src/WebJobs.Script/Content/Script/functions.js

```diff
@@ -33,11 +33,17 @@ function createFunction(f) {
 
         f = getEntryPoint(f, context);
 
-        var origLog = context.log;
-        context.log = function () {
-            value = util.format.apply(null, arguments);
-            origLog(value);
-        };
+        var loggers = context.log;
```

Comment:
I was thinking that you could construct the set of log methods in this code, but have them all funnel down into a single log call into the func exposed from .NET - just passing in an object of the form { level: 'info', message: '...' }. That allows us to just have a single simple method in .NET receiving all of these.